src/js/sources.js

```diff
@@ -191,34 +167,57 @@ export function ytcSource(window) {
     runQueue();
   };
 
-  const getTimestamp = (data, message) => data.isReplay
-    ? message.showtime
-    : (Date.now() + message.showtime) / 1000;
-
-  const extractToMsg = data => message => ({
-    timestamp: getTimestamp(data, message), message
+  const extractToMsg = () => (message) => ({
```

Comment:
Not quite sure why you used a ``() => (message) => {`` instead of a ``(message) => {``. This previously needed to be a factory function because it took in the data object for more context, but you seem to have refactored out the need for the data object.